resources/definitions/robotech3d_infinity_base.def.json

```
@@ -0,0 +1,147 @@
+        "speed_travel_layer_0":                         { "value": "math.ceil(speed_travel * 0.4)" },
+        "speed_layer_0":                                { "value": "math.ceil(speed_print * 0.25)" },
+        "speed_wall":                                   { "value": "math.ceil(speed_print * 0.33)" },
+        "speed_wall_0":                                 { "value": "math.ceil(speed_print * 0.33)" },
+        "speed_wall_x":                                 { "value": "math.ceil(speed_print * 0.66)" },
+        "speed_topbottom":                              { "value": "math.ceil(speed_print * 0.33)" },
+        "speed_roofing":                                { "value": "math.ceil(speed_print * 0.33)" },
+        "speed_slowdown_layers":                        { "default_value": 2 },
+        "roofing_layer_count":                          { "value": 1 },
+        "optimize_wall_printing_order":                 { "default_value": true },
+        "infill_enable_travel_optimization":            { "default_value": true },
+        "minimum_polygon_circumference":                { "default_value": 0.2 },
+        "wall_overhang_angle":                          { "default_value": 75 },
+        "wall_overhang_speed_factor":                   { "default_value": 50 },
+        "alternate_extra_perimeter":                    { "default_value": true },
+        "layer_height_0":                               { "resolve": "max(0.2, min(extruderValues('layer_height')))" },
+        "line_width":                                   { "value": "machine_nozzle_size * 1.125" },
+        "wall_line_width":                              { "value": "machine_nozzle_size" },
+        "fill_outline_gaps":                            { "default_value": true },
+        "meshfix_maximum_resolution":                   { "default_value": 0.01 },
+        "infill_before_walls":                          { "default_value": false },
+        "zig_zaggify_infill":                           { "value": true },
+        "min_infill_area":                              { "default_value": 5.0 },
+        "acceleration_enabled":                         { "default_value": false },
+        "acceleration_print":                           { "default_value": 2200 },
+        "acceleration_wall_0":                          { "value": 600 },
+        "acceleration_layer_0":                         { "value": 600 },
+        "acceleration_travel_layer_0":                  { "value": 600 },
+        "acceleration_roofing":                         { "value": 600 },
+        "jerk_enabled":                                 { "default_value": false },
+        "jerk_wall_0":                                  { "value": 10 },
+        "jerk_roofing":                                 { "value": 10 },
+        "cool_fan_speed":                               {"default_value": 25},
+        "infill_sparse_density":                        {"default_value": 10},
```

```suggestion
        "infill_sparse_density":                        {"default_value": 10}
```

Trailing , is not allowed in JSON. Please test the files in Cura first before asking for a review; This file won't load in Cura as it is now.

---

My mistake, I forgot to check for trailing comma's after editing it to fix the previous error